tests/sort_test.py

```diff
@@ -12,7 +12,26 @@ def testSort(self):
         spda = ak.sort(pda)
         maxIndex = spda.argmax()
         self.assertTrue(maxIndex > 0)
-   
+
+    def testBitBoundaryHardcode(self):
```

Comment:
It'd be good to add a test along these lines that includes `max(int)` and `min(int)` values

Reply:
(if there isn't already one -- I didn't look)

Reply:
added